bin/hr/hr.go

```diff
@@ -235,7 +241,16 @@ func (c *converter) transform(r io.Reader) {
 			}
 		}
 		if hrLine, err := c.formatter.Format(d); err == nil {
-			fmt.Println(hrLine)
+			if isatty(uintptr(syscall.Stdout)) {
+				// If in progress mode override infos in the same line
+				if priority == penlog.PrioInfo && c.showProgress {
+					fmt.Print("\033[2K" + hrLine + "\r")
```

Comment:
Not a fan of hardcoded escape codes. Let's rename the `color.go` file to `escapes.go` and add this one there. Maybe a function `eraseLine()` makes sense here.

Reply:
In principle II'm with you on this one.
I just didn't want to change any filenames or make a new one just for this single code.

Reply:
I figure there might come more escapes in the future. So let's do it.

Reply:
Ajo, let's prefer `fmt.Printf()` to `fmt.Print()` and concatenating strings.